cbmc_viewer/symbolt.py

```diff
@@ -173,6 +173,11 @@ class SymbolFromCtags(Symbol):
     list of files.
     """
 
+    # Warning: ctags by default accepts only certain file extensions
+    # like .c and .h as source files.  Files with other extensions
+    # like .inl are currently ignored, but consider using the ctags
```

Comment:
Couldn't we just replace the extension ".inl" with ".h" before invoking ctags?

Reply:
Replaced Warning with TODO.  This is just documenting the need for a solution.  I think using the flags to ctags described here is the most robust solution.